app/operations/application_operation.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationOperation
+  def self.call(*args)
+    new(*args).call
```

Comment:
Ok. operations. How should we divide services and operations? what will be the main difference?